mlir/test/Dialect/SparseTensor/sorted_coo.mlir

```diff
@@ -1,3 +1,4 @@
+// TODO: re-enable after lowering coo.next to function call (such that loop structure is more clear).
```

Comment:
Is this just because changing the CHECKs for now would be (too) much wasted effort?

Reply:
Yes, but also because I found the current CHECK test hard to read, it is difficult to tell whether the loop structure is correct.